doc/rst/platforms/aws.rst

```diff
@@ -162,7 +162,7 @@ Some other common optional configurations are:
     # Defaults to gethostname() of the launching node
     GASNET_MASTERIP='ec2-11-222-33-444.us-west-2.compute.amazonaws.com'
```

Comment:
I don't think we even need to mention `GASNET_MASTERIP` here, just replace it in with `CHPL_LAUNCH_MASTERIP` and maybe add a ref to `launcher.rst`.  (`GASNET_MASTERIP` is already covered in `launcher.rst` with a pointer to the GASNet docs and a note that it will be set by `CHPL_LAUNCH_MASTERIP`.)